src/odemis/acq/stream/_base.py

```diff
@@ -149,6 +153,15 @@ def __init__(self, name, detector, dataflow, emitter, focuser=None, opm=None,
         # side.
         self.auto_bc_outliers = model.FloatContinuous(100 / 256, range=(0, 40))
 
+        imageForHistogramAndDRange = None
+        if self.raw is not None:
+            if isinstance(self.raw, list):
+                if len(self.raw) > 0:
+                    imageForHistogramAndDRange = self.raw[0]
+            else:
```

Comment:
I realized that maybe it is a better idea to write to drange_raw only when the image is pyramidal. Do you agree?